### PR TITLE
Mrtk wireframe and standard shader fix

### PR DESCRIPTION
**Pull request by Alexees**
## Overview
Both shaders had errors, at least on Android.
Turned out both were missing one line of code mandatory for proper SinglePassInstancing usage:

https://docs.unity3d.com/Manual/SinglePassInstancing.html

```
v2f vert (appdata v)
{
    v2f o;

    UNITY_SETUP_INSTANCE_ID(v); //Insert
    UNITY_INITIALIZE_OUTPUT(v2f, o); //Insert
    UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o); //Insert

    o.vertex = UnityObjectToClipPos(v.vertex);

    o.uv = v.uv;

    return o;
}
```

## Changes
- Fixes: #5844

### CODE REVIEW

File: Assets/MixedRealityToolkit/StandardAssets/Shaders/MRTK_Wireframe.shader [hunk 1]
```diff
@@ -70,6 +70,7 @@ Shader "Mixed Reality Toolkit/Wireframe"
                 UNITY_SETUP_INSTANCE_ID(v);
                 v2g o;
                 o.viewPos = UnityObjectToClipPos(v.vertex);
+		UNITY_INITIALIZE_OUTPUT(v2g, o);
```
Comment (Alexees) [inline Assets/MixedRealityToolkit/StandardAssets/Shaders/MRTK_Wireframe.shader]: @Cameron-Micka this line fixed the error, but with my lack of knowledge about shaders and this one in particular, I can't say if this shouldn't go in line 71.

**Comment by Alexees** [comment]
wasn't quite it...

**Comment by Cameron-Micka** [comment]
Thank you for trying this! Did you still see the same shader compilation errors after adding these macros?

**Comment by Alexees** [comment]
@Cameron-Micka I'm not entirely sure. Too late I realized that changing the shader code could immediately remove the error in unity. Only when compiling the shader again via the compile button on the shader in the inspector the error when away.

**Pull request closed without merging**